exonum/src/blockchain/service.rs

```diff
@@ -169,9 +169,16 @@ pub trait Service: Send + Sync + 'static {
     /// [the `Service` example above](#examples).
     fn tx_from_raw(&self, raw: RawTransaction) -> Result<Box<dyn Transaction>, MessageError>;
 
-    /// Initializes the information schema of the service
-    /// and generates an initial service configuration.
-    /// This method is called on genesis block creation.
+    /// Invoked for all deployed services during the blockchain initialization
+    /// on genesis block creation at each time a node is started.
+    /// During the handling of the method the service able to perform the following activities:
+    /// - store its own initial state to the storage [`&mut Fork`]
+    /// - return an initial [global configuration][doc:global_cfg] of the service in the JSON
+    /// format, in case service has global configuration parameters. This configuration used
```

Comment:
`This configuration is used`. This one was mentioned before, but for some reason wasn't fixed.